src/main/java/Lotto/WinningLotto.java

```diff
@@ -0,0 +1,25 @@
+package lotto;
+
+import java.util.List;
+
+public class WinningLotto extends Lotto {
+    private LottoNumber bonusNumber;
+
+    public WinningLotto(List<LottoNumber> numbers, int bonusNumber) {
+        super(numbers);
+        this.bonusNumber = new LottoNumber(bonusNumber);
+        validateBonusNumber();
+    }
+
+    public int correctCount(Lotto myLotto) {
+        return (int) numbers.stream().filter(myLotto.numbers::contains).count();
```

Comment:
stream의 경우 개행을 하면 가독성에 도움이 될 것 같아요 😃 
```suggestion
        return (int) numbers.stream()
                .filter(myLotto.numbers::contains)
                .count();
```